test/Microsoft.Azure.Devices.Edge.Test.Common/DaemonConfiguration.cs

```diff
@@ -12,11 +12,24 @@ public class DaemonConfiguration
         readonly string configYamlFile;
         readonly YamlDocument config;
 
-        public DaemonConfiguration(string configYamlFile)
+        public DaemonConfiguration(string configYamlFile, (string image, string address, string username, string password) bootstrapAgentInfo)
```

Comment:
This also means that `CreateEdgeDaemon` and the constructors for linux and windows `EdgeDaemon` classes would take two new params instead of one, and both would be optional with defaults.